go/vt/mysqlctl/mysqld.go

```diff
@@ -1293,6 +1294,68 @@ func (mysqld *Mysqld) GetVersionComment(ctx context.Context) (string, error) {
 	return res.ToString("@@global.version_comment")
 }
 
+// systemMetrics returns several OS metrics to be used by the tablet throttler.
+func systemMetrics(ctx context.Context, cnf *Mycnf) (*mysqlctlpb.SystemMetricsResponse, error) {
+	resp := &mysqlctlpb.SystemMetricsResponse{
+		Metrics: make(map[string]*mysqlctlpb.SystemMetricsResponse_Metric),
+	}
+	newMetric := func(name string) *mysqlctlpb.SystemMetricsResponse_Metric {
+		metric := &mysqlctlpb.SystemMetricsResponse_Metric{
+			Name: name,
+		}
+		resp.Metrics[name] = metric
+		return metric
+	}
+	withError := func(metric *mysqlctlpb.SystemMetricsResponse_Metric, err error) error {
+		if err != nil {
+			metric.Error = err.Error()
+		}
+		return err
+	}
+
+	_ = func() error {
+		metric := newMetric("datadir-used")
```

Comment:
This name is not obvious to me. IMO `pct-datadir-used` is clearer.

Reply:
Renamed to `datadir-used-ratio` (The value is `[0.0..1.0]`).